client/client.ts

```diff
@@ -342,7 +356,20 @@ async function tryToSubmitMove() {
     );
 
     endProveTime = Date.now();
-    console.log(`Time to prove: ${endProveTime - startProveTime}ms`);
+    const provingTime = endProveTime - startProveTime;
```

Comment:
Could you create a new function provingtime logic

Reply:
wdym by a new function for elapsed time?